docs/users.md

```diff
@@ -45,9 +45,15 @@ rs0:PRIMARY> db.test.findOne()
 To automate the deployment and management of the cluster components, 
 the Operator requires system-level MongoDB users.
 
-During installation, the Operator requires Kubernetes Secrets to be deployed
-before the Operator is started. The name of the required secrets can be set in
+Credentials for these users are stored as a [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) object.
+The Operator requires Kubernetes Secrets before the database cluster is
```

Comment:
it's a bit strange using of Secrets as a plural, object `kind` is `Secret` although when you query you can use both `secret` and `secrets`